AffiliatedPackages.md

```diff
@@ -0,0 +1,111 @@
+reuse, interoperability, and interface standards for Python planetary
+science packages. In many (but not all) cases, affiliated
+packages also follow similar devlopment processes and package
+templates as for the core package.
+
+PlanetaryPy affiliated packages are autonomous projects created and 
+managed by their own Technical Committees (TC), not the PlanetaryPy TC.
+
+These are primarily software projects of various kinds which already
+exist separately.  They benefit from being PlanetaryPy affiliated packages
+in the following ways: 
+
+1. The PlanetaryPy TC helps them adopt or formalize governance and
+	community standards to foster their existance as an open
+	source community.
+2. They gain the ability to easily reach out to the Planetary
+	Software TC for consultation or mentorship on technical
+	or governance issues.
+3. Being recognized as a PlanetaryPy affiliated package lets others
+	know that their project conforms to community best practices
+	which are likely to result in long-term success for their
+	project.
+
+## Becoming an Affiliated Package
+
+If you are a developer of a planetary science package and would like your
+package to become affiliated with the PlanetaryPy Project, please take
+a look at the [instructions for proposing an affiliated package][review-process]. We
+recommend that you also take a look at the [guidelines for reviewing
+affiliated packages][guidelines] since this will give you a sense of whether
+your package is ready for review. Broadly speaking, your package
+should:
+
+* Be potentially useful to planetary scientists. This can mean
+  useful to a specific sub-domain of planetary science, or more broadly useful
+  to a large fraction of planetary science (or beyond, as long as it is also
+  useful for planetary science).
+
+* Be written in a way that is readable and understandable by others. While 
+  not a strict requirement, we also provide coding guidelines that will make 
+  your code easier to read by members of the community.
+
+* Use classes and functions from the `planetarypy` core package wherever possible
+  and appropriate, and (as much as possible) avoid duplication with other packages
+  in the PlanetaryPy ecosystem. This facilitates re-use of code and sharing of resources.
+
+* Have documentation that adequately explains the use of the package. Additionally,
+  user-facing classes and functions should all have docstrings. We suggest using 
+  sphinx for your documentation.
+
+* Make a best-effort to include an easy-to-run test suite that covers its intended
+  functionality. We realize this is not always possible, but when it is, a test 
+  suite is a crucial element of stable software and reproducible science.
+
+* Be compatible with Python 3.6 and above.
+
+* Be open to contributions from others. While this most straightforwardly means it 
+  follows a Github-based open development model (like the PlanetaryPy core package),
```

Comment:
means *that* it follows? (i would have put that *that* there)...

Reply:
Yeah, that's a terrible sentence that I copied.  I changed it around more than just one word, and the end was weird, too.  See if you like it better in the upcoming commit.